App.css

```diff
@@ -0,0 +1,79 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.header {
+  background-color: #3c3a39;
+}
+
+.nav {
+  display: flex;
+  height: 3rem;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem;
+}
+
+.nav-title {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  color: #fff;
+  font-size: 1rem;
+  padding-left: 0.75rem;
+  padding-bottom: 0.625rem;
+}
+
+.nav-menu {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+
+.hero {
+  background: #1c1a19 url("../images/Illustration.svg") 100% 100% no-repeat;
+}
+
+.main {
+  height: 36.875rem;
+  left: 0;
+  top: 53px;
+  padding: 120px 24px 120px 24px;
+}
+
+.main-header {
+  font-family: 'Crete Round', serif;
+  font-size: 3.5rem;
+  color: #ff6b00;
+}
+
+.sub-header {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  color: #fff5e1;
+  font-size: 1.5rem;
+}
+
+.main-paragraph {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  color: #fff;
+}
+
+.social-media {
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+}
+
+.github,
+.linkedin,
+.angelist,
+.twitter,
+.medium {
+  height: 20px;
```

Comment:
- [ ] Kindly work on the size of social icons it looks so small.
<img width="314" alt="Screenshot 2023-02-17 at 07 41 02" src="https://user-images.githubusercontent.com/81444338/219699741-4aa07c37-e527-4ee3-af11-0917df56c2bf.png">

Reply:
@Kossi-stack 
Hello,
I'm not understanding the second option suggestion. Can you help me?